src/core/jit/response.rs

```diff
@@ -19,3 +25,13 @@ impl<Value, Error> Response<Value, Error> {
         }
     }
 }
+
+impl Response<async_graphql::Value, jit::Error> {
+    pub fn try_into_hyper_response(self) -> anyhow::Result<hyper::Response<hyper::Body>> {
+        let body = serde_json::to_string(&self)?;
+        let resp = hyper::Response::builder()
+            .header("Content-Type", "application/json")
```

Comment:
Other general purpose headers must be added from the request handler, we can just create a basic hyper response.